spec/safe_statements_spec.rb

```diff
@@ -1,18 +1,26 @@
 require "spec_helper"
 
 RSpec.describe PgHaMigrations::SafeStatements do
-  TableLock = Struct.new(:table, :lock_type, :granted)
+  TableLock = Struct.new(:table, :lock_type, :granted, :pid)
   def locks_for_table(table, connection:)
+    identifiers = table.to_s.split(".")
+
+    identifiers.prepend("public") if identifiers.size == 1
+
+    schema, table = identifiers
+
     values = connection.execute(<<-SQL)
```

Comment:
I think we can now use this?
```suggestion
    values = connection.structs_from_sql(<<-SQL)
```

Note: this could totally be a separate PR.

Reply:
Addressed in https://github.com/braintree/pg_ha_migrations/pull/87/commits/bf4407d629c2b4eb8e5f1d23f85f9a8087e8fd06